.github/workflows/release-internal.yml

```diff
@@ -8,9 +20,10 @@ on:
         type: string
         required: true
       sha:
```

Comment:
I know the team went over this so ignore me if you all agreed but you should call this ref if it's a ref and not always a sha.  Esp since this is the name you will see when manually submitting the workflow to release.